<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="7">
     <widget class="QTextEdit" name="previousNote">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhMultiLine</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="7">
     <widget class="QTextEdit" name="currentNote">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhMultiLine</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="7">
     <widget class="QTextEdit" name="nextNote">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhMultiLine</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="everyFeelingBox">
      <property name="text">
       <string>Wszystkie</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QCheckBox" name="sadFeelingBox">
      <property name="text">
       <string>Smutne</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QCheckBox" name="happyFeelingBox">
      <property name="text">
       <string>Radosne</string>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>183</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>176</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QPushButton" name="deleteNote">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Usun wpis</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QToolButton" name="upButton">
      <property name="text">
       <string>...</string>
      </property>
      <property name="arrowType">
       <enum>Qt::UpArrow</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="4" colspan="2">
     <widget class="QPushButton" name="loadBackUpButton">
      <property name="text">
       <string>Wczytaj kopię zapasową</string>
      </property>
     </widget>
    </item>
    <item row="4" column="6">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>183</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>176</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="1" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Dodaj wpis</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QToolButton" name="downButton">
      <property name="text">
       <string>...</string>
      </property>
      <property name="arrowType">
       <enum>Qt::DownArrow</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="4" colspan="2">
     <widget class="QPushButton" name="saveBackUpButton">
      <property name="text">
       <string>Zapisz kopię zapasową</string>
      </property>
     </widget>
    </item>
    <item row="5" column="6">
     <spacer name="horizontalSpacer_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>183</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="0">
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>183</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="2" colspan="3">
     <widget class="QPushButton" name="pushSort">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Sortuj</string>
      </property>
     </widget>
    </item>
    <item row="6" column="6">
     <spacer name="horizontalSpacer_8">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>183</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="3" colspan="2">
     <widget class="QCheckBox" name="neutralFeelingBox">
      <property name="text">
       <string>Neutralne</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
